_posts/2017-03-16-Rust-1.16.md

```diff
@@ -201,11 +201,6 @@ results in a significant improvement in these sorts of diagnostics.
 [This now happens nearly everywhere]: https://github.com/rust-lang/rust/pull/38927
 [some other related improvements]: https://github.com/rust-lang/rust/pull/38154
 
-Finally, `Self` can now appear [in impl headers] and [in struct expressions].
```

Comment:
@steveklabnik 
My comment was only about impl headers, `Self` in struct expressions (`Self { a: 10 }`) is fully supported in 1.16